src/impl/tpls/KokkosBlas2_gemv_tpl_spec_decl.hpp

```diff
@@ -457,7 +457,7 @@ struct GEMV< \
     \
     bool A_is_lr = std::is_same<Kokkos::LayoutRight,LAYOUTA>::value; \
     A.stride(strides); \
```

Comment:
@mhoemmen I think yes, but aren't two strides are still needed?

Reply:
Yes -- I was just thinking it would be nicer to get the first two strides by value, rather than using this old "write results to an array" interface.  It doesn't matter, though.